src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    return movie.title.toLowerCase().includes(query.toLowerCase())
+    || movie.description.toLowerCase().includes(query.toLowerCase());
```

Comment:
Good work, but it will be more readable if you will create separate variables, and compare them, because at now it looks hard)) 
Shortest is not the best in every time)

```suggestion
    const titleInclude = movie.title.toLowerCase().includes(query.toLowerCase());
    const decriptionInclude = movie.description.toLowerCase().includes(query.toLowerCase());
    
    return titleInclude || decriptionInclude;
```